Makefile

```diff
@@ -29,7 +29,7 @@ clean:
 
 test:
 	@echo "*** Running unittests ***"
-	PYTHONPATH=. $(PYTHON) -m unittest discover -v -s tests/ -p '*_test.py'
+	$(PYTHON) -m unittest discover -v -s tests/ -p '*_test.py'
```

Comment:
What will happen if I have `simpleline` installed on my system and I need to test changes in my git repository? Which `simpleline` will be tested?

Reply:
PYTHONPATH="." is not meaningful because PYTHONPATH is always set to '.' by default and that is the first place where it will look for sources. This is just a simplification. So yes, running `make test` will work as expected but running this line in the tests folder will run the tests on installed library in the system.

Reply:
https://docs.python.org/3/library/sys.html#sys.path
```